### PR TITLE
MotorModule prepared for merge and integration

### PR DESCRIPTION
**Pull request by ThunFjsch**
The MotorModule feature is as far implemented as possible. Next it needs to be Integrated with other Modules.

### CODE REVIEW

File: MotorModule.cpp [hunk 1]
```diff
@@ -0,0 +1,210 @@
+#include <time.h>
```
Comment (leobeo-png) [inline MotorModule.cpp:1]: No need to include again, since it's already in the header.

File: MotorModule.cpp [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+#include <Arduino.h>
```
Comment (leobeo-png) [inline MotorModule.cpp:2]: No need to include time.h and Arduino.h, since it was already included in the header file.

Comment (leobeo-png) [reply MotorModule.cpp:2]: Other than that, all other files seems good.

**Review by leobeo-png — APPROVED** [review]
nice.

**Review by DrTowfiq-666 — APPROVED** [review]

**Pull request merged**